int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -489,7 +472,22 @@ function setCheckoutFormEvents() {
 	* the listeners are set to refresh the configuration for Global Intuitive.
 	* For more information see Bug #125898 */
 	const setEventsForRegistration = function() { setEventsForListeners(); };
-	const setEventsForBilling = function() { setEventsForListeners("billing"); };
+	const setEventsForBilling = function() { 
+		setEventsForListeners("billing");
+		if (reloadGIjs) {
```

Comment:
Added the variable called "reloadGIjs" (boolean) to reload only once when the user switch between shipping and billing on the checkout stage and moved the Global Intuitive re-loading since is the same website controlled by CSS and JavaScript the website isn't reloading the Globa Intuitive js; with this the Global Intuitive js will add the mapping and Suggested Address box to both fields (shipping and billing).

Reply:
This is fine, but another way you could've done this is just by using the state of the HTML to determine which (shipping or billing) mode the user is in instead of using a variable.